### DISCUSSION TITLE
Wunsch / Wish: Zusammenfassung / Ad summary

### DISCUSSION THREAD

**Discussion by LinuxOpa** [cboxdoerfer/fsearch#445] [Ideas]
Ein Wunsch / a wish

statt angezeigt / instead of displayed:

/daily.0/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
/daily.1/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
/daily.2/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
/daily.3/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
/daily.4/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
/daily.5/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
/daily.6/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg

zusammenfassen und erst nach anklicken aufklappen / summarize and expand only after click:

[8] Austin_FX4Q_London_Taxi.jpg

klick / click:

[8] Austin_FX4Q_London_Taxi.jpg:
->  /daily.0/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
->  /daily.1/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
->  /daily.2/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
->  /daily.3/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
->  /daily.4/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
->  /daily.5/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
->  /daily.6/home/user/Bilder/Austin_FX4Q_London_Taxi.jpg
->  /home/user/Bilder/Austin_FX4Q_London_Taxi.

**Comment by cboxdoerfer** [comment]
That's an interesting idea. So it's grouping the results by their filename. So what do you expect to happen when you don't sort by name, but by size? For example:

| Name ^ | Path | Size |
|----------|---------|-------|
pic1.jpg    |   /backup/pictures | 1MB
pic1.jpg    |   /home/test/downloads | 5MB
pic1.jpg    |   /home/test/pictures | 1MB
pic2.jpg    |   /backup/pictures | 3MB
pic2.jpg    |   /home/test/downloads | 5MB
pic2.jpg    |   /home/test/pictures | 3MB

would look like this when grouped:

| Name ^ | Path | Size |
|----------|---------|-------|
\> pic1.jpg (3 Items) | ... | ...
\> pic2.jpg (3 Items) | ... | ...

But what about this situation, when the list is sorted by size instead?

| Name | Path | Size ^ |
|----------|---------|-------|
pic1.jpg    |   /backup/pictures | 1MB
pic1.jpg    |   /home/test/pictures | 1MB
pic2.jpg    |   /backup/pictures | 3MB
pic2.jpg    |   /home/test/pictures | 3MB
pic1.jpg    |   /home/test/downloads | 5MB
pic2.jpg    |   /home/test/downloads | 5MB

Should it then also group by size, like so:

| Name | Path | Size ^ |
|----------|---------|-------|
\>  1MB (2 Items) | ... | ...
\>  3MB (2 Items) | ... | ...
\>  5MB (2 Items) | ... | ...

Or still by name, which might look like this:

| Name | Path | Size ^ |
|----------|---------|-------|
\> pic1.jpg  (2 Items)  |   ... | ...
\> pic2.jpg (2 Items)   | ... | ...
pic2.jpg    |   /home/test/pictures | 3MB
pic1.jpg    |   /home/test/downloads | 5MB
pic2.jpg    |   /home/test/downloads | 5MB

Or something else?

**Reply by LinuxOpa** [reply]
All possible combinations are useful.

**Comment by LinuxOpa** [comment]
**Bitte SPAM oben löschen - please delete SPAM above - ist nicht von mir - is not mine!**
